hi-ml-histopathology/testhisto/testhisto/mocks/utils.py

```diff
@@ -0,0 +1,25 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import logging
+from pathlib import Path
+
+from health_azure.datasets import DatasetConfig
+
+from health_azure.utils import WORKSPACE_CONFIG_JSON, create_config_json, get_workspace
+from testazure.utils_testazure import get_shared_config_json
+
+
+def download_azure_dataset(tmp_path: Path, dataset_id: str) -> None:
```

Comment:
DRY (don't repeat yourself) - how much duplication is there between this fuction and mount_mnist_dataset?